File: Dart/Slice/slice_length.dart
import "dart:core";

void main() {
  // Create a new list of strings.
  final List<String> randomSlice = <String>["Bob", "Alice", "Joe"];
  // Get the length of the slice.
  print(getSliceLength(randomSlice));
}

// Get the length of a given slice.
int getSliceLength(final List<dynamic> slice) {
  return slice.length;
}

File: TypeScript/Slice/slice-length.ts
function main(): void {
    // Create a slice of random strings.
    const randomSlice: string[] = ["a", "b", "c", "d", "e"]
    // Get the length of the slice.
    console.log(getSliceLength(randomSlice))
}

main()

// Get the length of a given slice.
function getSliceLength(originalSlice: string[]): number {
    return originalSlice.length
}